stdlib/task/src/main/ballerina/task/Module.md

```diff
@@ -1,18 +1,18 @@
 ## Module overview
 
-This module includes functions to manage Task Listeners and Task Schedulers.
+This module provides functionality to configure and manage Task Listeners and Task Schedulers, which will be executed periodically.
```

Comment:
"which will be executed periodically." <- What is meant by this? What does it mean to execute a task listener or a task scheduler periodically?